src/Cli/Command.php

```diff
@@ -211,9 +211,9 @@ protected function handleMessage($id, $message, $ttr, $attempt)
 
             return $result === self::EXEC_DONE;
         } catch (ProcessRuntimeException $error) {
-            list($job) = $this->queue->unserializeMessage($message);
+            [$job] = $this->queue->unserializeMessage($message);
 
-            return $this->queue->handleError(ExecEvent::before($id, $job, $ttr, $attempt, $error));
+            return $this->queue->handleError(ExecEvent::error($id, $job, $ttr, $attempt, $error));
```

Comment:
This is wrong, error() takes an Event as first and only parameter.

@zhuravljov something went horribly wrong with ErrorEvent removal, and that leads to "retry" tests failing.
I'm having a little trouble debugging this as this is all happening in child processes.. could you have a look please? 🙏